### PR TITLE
Hiding of not active courses (fixes #47)

### LINKED ISSUE #47: Course management
There should be some kind of course management. Now one can only create a course, but cannot delete it. Hiding not active courses will also be useful (or displaying them alternatively).

### PR DESCRIPTION
**Pull request by Dervun**
> Hiding not active courses will also be useful (or displaying them alternatively).

### CODE REVIEW

File: app/views/courses/_index.html.erb [hunk 1]
```diff
@@ -1,5 +1,26 @@
...
+      </li>
+    </ul>
+  </div>
+</div>
+<br/>
+<div class="row">
+  <div class="col-md-12">
+    <div class="tab-content">
+      <div id="active" class="tab-pane fade in active">
+				<%= render 'courses/courses', { courses: @courses[:active] } %>
```
Comment (auduchinok) [inline app/views/courses/_index.html.erb]: Format this line, please.

**Comment by Dervun** [comment]
Strangely, but there was such indentation

**Comment by auduchinok** [comment]
There's also a request for ability to close course in #47. Any ideas?

**Comment by auduchinok** [comment]
Let's place a button for closing a course on a top right and remove teacher's name and email when a teacher is visiting the page?
screen shot 2016-12-05 at 23 44 26

**Comment by Dervun** [comment]
> Now one can only create a course, but cannot delete it.
It's #99 
And about

> Let's place a button for closing a course on a top right and remove teacher's name and email when a teacher is visiting the page?
I will make it in #99 too.

**Pull request merged**